docs/architecture/kvbm_components.md

```diff
@@ -173,11 +173,11 @@ The left side of the figure in [Understanding KVBM Components](#understanding-kv
        The system can batch and publish registration events using a Publisher, optimizing performance under high concurrency
 
 
-#### Storage backends and pluggability
+## Storage backends and pluggability
 
-You can integrating KVBM with a storage backend by extending or wrapping `NixlEnabledStorage` to support cross-node RDMA registration. All layouts and block pools are generic over these backends, allowing for fine-grained control over memory tiers. We defer detailed integration guidance, since we collaborate with storage partners to simplify and standardize these integration paths.
+You can integrate KVBM with a storage backend by extending or wrapping `NixlEnabledStorage` to support cross-node RDMA registration. All layouts and block pools are generic over these backends, allowing for fine-grained control over memory tiers. We defer detailed integration guidance, since we collaborate with storage partners to simplify and standardize these integration paths.
 
-```
+```none
 An example system architecture
```

Comment:
Any chance this could be in a mermaid chart? If it takes too much time no need to bother :)